### DISCUSSION TITLE
[Bug]: `Image.fromObject(imgEl)` copy all properties and methods from `imgEl` and contains `contains` method from 'imgEl`

### DISCUSSION THREAD

**Discussion by ruojianll** [fabricjs/fabric.js#8548] [Q&A] [answered]
### CheckList

- [X] I agree to follow this project's Code of Conduct
- [X] I have read and followed the [Contributing Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md)
- [X] I have read and followed the [Issue Tracker Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md#%EF%B8%8F-issue-tracker)
- [X] I have searched and referenced existing issues and discussions
- [X] I am filing a **BUG** report.
- [X] I have managed to reproduce the bug **after** upgrading to the **latest** version
- [X] I have created an accurate and minimal reproduction

### Version

5.3.0

### In What environments are you experiencing the problem?

Chrome

### Node Version (if applicable)

None

### Link To Reproduction

https://jsfiddle.net/95azs41o/9/

### Steps To Reproduce

1. Create a image object `oImg` from html img element.
2. Add `oImg` object in to canvas.
3. Create another object `const rect = fabric.Rect({})`
4. Call `fabric.contains(rect,true)`

### Expected Behavior

Return `false` and no error occurred.

### Actual Behavior

Throw error
`Uncaught TypeError: Illegal invocation`

### Error Message & Stack Trace

```bash
fabric.js:509 Uncaught TypeError: Illegal invocation
    at fabric.js:509:58
    at Array.some (<anonymous>)
    at klass.contains (fabric.js:508:28)
    at ?editor_console=:138:10
    at fabric.js:21607:13
    at fabric.js:1221:21
    at Object.enlivenObjects (fabric.js:1187:21)
    at Object.enlivenObjectEnlivables (fabric.js:1215:19)
    at fabric.js:21605:23
    at fabric.js:21487:23
```

**Comment by ShaMan123** [comment, marked as answer]
https://jsfiddle.net/y03xm679/

The error is correct. You misused.

**Reply by ruojianll** [reply]
Is `fromObject` extending a object and use it's src as url?
